Lib/test/string_tests.py

```diff
@@ -80,12 +80,14 @@ class subtype(self.__class__.type2test):
                 self.assertIsNot(obj, realresult)
 
     # check that obj.method(*args) raises exc
-    def checkraises(self, exc, obj, methodname, *args):
+    def checkraises(self, exc, obj, methodname, *args, **kwargs):
```

Comment:
You can just put `expected_msg=None` here I believe.

Reply:
Done! That's better indeed.